workflow-dockstore-metrics/workflow_dockstore_metrics/pull_and_send_metrics.py

```diff
@@ -0,0 +1,202 @@
+"""
+This script pulls workflow execution data from BigQuery and sends it to Dockstore.
```

Comment:
Nit, to avoid diverging, you could simply have the header of this file refer to the README.md file since the content is exactly the same